pkg/commands/compute/init_test.go

```diff
@@ -187,6 +232,32 @@ func TestInit(t *testing.T) {
 				"SUCCESS: Initialized package",
 			},
 		},
+		{
+			name: "with --from set to zip archive when file with same name exists in pwd",
+			args: args("compute init --auto-yes --from https://github.com/fastly/compute-starter-kit-rust-default/archive/refs/heads/main.zip"),
+			configFile: config.File{
+				StarterKits: config.StarterKitLanguages{
+					Rust: []config.StarterKit{
+						{
+							Name: "Default",
+							Path: "https://github.com/fastly/compute-starter-kit-rust-default.git",
+						},
+					},
+				},
+			},
+			wantOutput: []string{
+				"Fetching package template",
+				"Reading fastly.toml",
+				"SUCCESS: Initialized package",
+			},
+			setupSteps: func() error {
+				file, err := os.Create("main.zip")
+				if file != nil {
```

Comment:
```suggestion
				if file, err := os.Create("main.zip"); file != nil {
```

Reply:
I had to revert this one because `err` was needed outside the scope of the `if` block.